cogs/tags/tags.py

```diff
@@ -200,13 +205,14 @@ async def user_exceeds_tag_limit(self, server: discord.Guild, user: discord.Memb
 
         Returns:
         --------
-        bool
-            True if too many tags, else False.
+        (bool, int)
+            bool: True if user has too many tags, else False.
+            int: The maximum number of tags this user can have. -1 if they
+            have no limit.
         """
-
         if await self.bot.is_owner(user):
             # No limit for bot owner
```

Comment:
Minor, but if the owner wants to have a limit they can't

Reply:
Will disregard for now, since this exists already.